enclone_core/src/combine_group_pics.rs

```diff
@@ -22,8 +22,8 @@ pub fn combine_group_pics(
     if noprint && parseable_stdouth && !group_pics.is_empty() {
         let mut rows = Vec::<Vec<String>>::new();
         for pic in group_pics {
-            let r = pic.split('\n');
-            for rj in r.skip(1) {
+            let r: Vec<_> = pic.split('\n').collect();
+            for rj in r.iter().take(r.len() - 1) {
                 let s = rj.split('\t').map(str::to_owned).collect();
```

Comment:
Also not touched here but this to_owned seems totally unnecessary; `rows` doesn't outlive this `if` block, let alone `group_pics`
```suggestion
                let s = rj.split('\t').collect();
```
And the collect is also unnecessary.  So much useless allocation.

Reply:
This change results in a cascade of other changes. Won't handle them in this PR